examples/hitl/rearrange_v2/config/task.yaml

```diff
@@ -0,0 +1,87 @@
+# @package _global_
```

Comment:
Similar requests for this config:
* add a comment explaining what Habitat-lab config this was derived from.
* name is something meaningful. "task" is vague. Maybe this is a specific task, e.g. instruction-following or social rearrange?
* strip it down to the essential config fields. E.g. you name a `dataset.data_path` here that I don't think actually gets used.